client/package.json

```diff
@@ -7,9 +7,11 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "axios": "^0.20.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.0"
+    "react-scripts": "3.4.0",
+    "typescript": "^4.0.3"
```

Comment:
Looks like you added TS and a config but no ts-node, tsc, etc to your scripts. Make sure you want to use TypeScript before setting it up, or just remove this dependency.

Reply:
Ah, yep. Had it in there to fix an error. Just changed the BlogForm.test.tsx to .jsx and removed typescript and config. All good now.